x-pack/plugins/alerting/server/routes/get_query_delay_settings.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { ILicenseState } from '../lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+import { verifyAccessAndContext, RewriteResponseCase } from './lib';
+import { API_PRIVILEGES, RulesSettingsQueryDelay } from '../../common';
+
+const rewriteBodyRes: RewriteResponseCase<RulesSettingsQueryDelay> = ({
+  createdBy,
+  updatedBy,
+  createdAt,
+  updatedAt,
+  ...rest
+}) => ({
+  ...rest,
+  created_by: createdBy,
+  updated_by: updatedBy,
+  created_at: createdAt,
+  updated_at: updatedAt,
+});
+
+export const getQueryDelaySettingsRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.get(
+    {
+      path: `${INTERNAL_BASE_ALERTING_API_PATH}/rules/settings/_query_delay`,
```

Comment:
I created rule settings on `main` and then switched to this branch and then accessed this route and got a 500 error

Reply:
Resolved in this commit [dc7bbc8](https://github.com/elastic/kibana/pull/167433/commits/dc7bbc80b23f7dcd43d391cdd73844daeee0abb9)